packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1368,10 +1368,9 @@ class Transform extends SingleChildRenderObjectWidget {
   }) : transform = Matrix4.translationValues(offset.dx, offset.dy, 0.0),
        origin = null,
        alignment = null;
-
   /// Creates a widget that scales its child along the 2D plane.
   ///
-  /// The `scaleX` argument provides the scalar by which to multiply the `x` axis, and the `scaleY` argument provides the scalar by which to multiply the `y` axis. Either may be omitted, in which case that axis defaults to 1.0.
+  /// The `scaleX` argument provides the scalar by which to multiply the `x` axis, and the `scaleY` argument provides the scalar by which to multiply the `y` axis. Either may be omitted, in which case the scaling factor for that axis defaults to 1.0.
```

Comment:
This looks good to me.  If you want to remove the remaining ambiguity, one suggestion would be "Either may be omitted, in which case the scaling factor for the omitted axis defaults to 1.0.," but now that I write that out I feel like it's probably going too far.  Your call, but I'd be happy with you submitting this to Flutter as you wrote it.